build.sbt

```diff
@@ -2,13 +2,14 @@ import sbt.Keys._
 import sbt._
 // shadow sbt-scalajs' crossProject and CrossType from Scala.js 0.6.x
 import sbtcrossproject.CrossPlugin.autoImport.{crossProject, CrossType}
-
-ThisBuild / scalafmtOnCompile := true
```

Comment:
I had to disable scalafmt on compile because of a bug scalameta/scalafmt#2722.

Reply:
The problem was not on scalafmt's side, I've fixed the configuration and brought back `scalafmtOnCompile := true`